opennms-alarms/daemon/src/test/java/org/opennms/netmgt/alarmd/AlarmdIT.java

```diff
@@ -233,6 +235,34 @@ public void testPersistAlarm() throws Exception {
         }
     }
     
+    @Test
+    public void canHandleMultipleClearsWithNoCorrrespondingProblemAlarm() throws Exception {
+        final MockNode node = m_mockNetwork.getNode(1);
+        sendNodeUpEvent(node);
+        sendNodeUpEvent(node);
+
+        // We should only have one alarm now and should have a count of 2
+        await().until(() -> m_alarmDao.findAll().size() == 1 && m_alarmDao.findAll().get(0).getCounter() == 2);
```

Comment:
This should be broken out into multiple assertions - if it fails which one didn't match?